index.js

```diff
@@ -3,12 +3,13 @@ var cli = require('./libs/cli');
 
 program
   .version('0.0.1')
+  .option('-d, --directions', 'Repeat the current challenge directions', cli.directions)
```

Comment:
The instructions stated that `gitfun` would repeat the directions, but that is only true for levels that don't require input. I added an explicit `gitfun directions` option. I was tempted to call it `gitfun repeat`…